packages/i18n-calypso/test/index.js

```diff
@@ -0,0 +1,376 @@
+			it( 'should know the short weekdays', function() {
+				assert.equal( 'Fr', moment( '2014-07-18' ).format( 'dd' ) );
+			} );
+			it( 'should use available translations for date format', function() {
+				assert.equal( 'Freitag, 18. Juli 2014 21:59', moment( '2014-07-18T14:59:09-07:00' ).utcOffset( '+00:00' ).format( 'LLLL' ) );
+			} );
+			it( 'should use available translations for relative time in the past', function() {
+				assert.equal( 'vor 3 Stunden', moment().subtract( 3, 'hours' ).fromNow() );
+			} );
+			it( 'should use available translations for relative time in the future', function() {
+				assert.equal( 'in ein paar Sekunden', moment().add( 10, 'seconds' ).fromNow() );
+			} );
+			it( 'should be able to convert dates to any timezone', function() {
+				assert.equal( 'Freitag, 18. Juli 2014 14:59', moment( '2014-07-18T14:59:09-07:00' ).tz( 'America/Los_Angeles' ).format( 'LLLL' ) );
+				assert.equal( 'Samstag, 19. Juli 2014 06:59', moment( '2014-07-18T14:59:09-07:00' ).tz( 'Asia/Tokyo' ).format( 'LLLL' ) );
+				assert.equal( 'Freitag, 18. Juli 2014 23:59', moment( '2014-07-18T14:59:09-07:00' ).tz( 'Europe/Paris' ).format( 'LLLL' ) );
+				assert.equal( 'Freitag, 18. Juli 2014 22:59', moment( '2014-07-18T14:59:09-07:00' ).tz( 'Europe/London' ).format( 'LLLL' ) );
+			} );
+		} );
+	} );
+
+	describe( 'numberFormat()', function() {
+		describe( 'default numberFormat', function() {
+			it( 'should truncate decimals', function() {
+				assert.equal( '150', numberFormat( 150.15 ) );
+			} );
+			it( 'should round up', function() {
+				assert.equal( '151', numberFormat( 150.5 ) );
+			} );
+			it( 'should default to locale thousands separator (. for German in test)', function() {
+				assert.equal( '1.500', numberFormat( 1500 ) );
+			} );
+		} );
+
+		describe( 'with decimal', function() {
+			it( 'should default to locale decimal separator (, for German in test)', function() {
+				assert.equal( '150,00', numberFormat( 150, 2 ) );
+			} );
+			it( 'should truncate to specified decimal', function() {
+				assert.equal( '150,31', numberFormat( 150.312, 2 ) );
+			} );
+			it( 'should accept decimal as argument or object attribute', function() {
+				assert.equal( '150,00', numberFormat( 150, {
+					decimals: 2
+				} ) );
+			} );
+		} );
+
+		describe( 'overriding defaults', function() {
+			it( 'should allow overriding of locale decimal and thousands separators', function() {
+				assert.equal( '2*500@330', numberFormat( 2500.33, {
+					decimals: 3,
+					thousandsSep: '*',
+					decPoint: '@'
+				} ) );
+			} );
+		} );
+	} );
+
+	describe( 'hashed locale data', function() {
+		it( 'should find keys when looked up by simple hash', function() {
+			i18n.setLocale( {
+				'': {
+					localeSlug: 'xx-pig-latin',
+					'key-hash': 'sha1'
+				},
+				'0f7d0d088b6ea936fb25b477722d734706fe8b40': [
+					null,
+					'implesa'
+				]
+			});
+			assert.equal( i18n.translate( 'simple' ), 'implesa' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **35** times:
`translate( 'Simple', { context: 'Theme Showcase term name'} )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).